src/components/Note.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+
+class Note extends Component {
+
+	render() {
+		return (
+			<div className="note-div">
```

Comment:
Could improve classname to be a bit more meaningful, 'note-content'? 'note-container'?